packages/replay/src/install.ts

```diff
@@ -20,6 +21,9 @@ function getBrowserDownloadFileName<K extends keyof typeof EXECUTABLE_PATHS>(key
   switch (key) {
     case "darwin:firefox":
       return process.env.RECORD_REPLAY_FIREFOX_DOWNLOAD_FILE || "macOS-replay-playwright.tar.xz";
+    case "darwin:chromium":
+      return process.env.RECORD_REPLAY_CHROMIUM_DOWNLOAD_FILE || "macOS-replay-chromium.tar.xz";
```

Comment:
we might want to extend this to also support Chrome for ARM which can be downloaded as well and would be faster